[ 2022-05-02T10:32:24     INFO ] logging to applications insights
[ 2022-05-02T10:32:25     INFO ] [ exists                   ] True
[ 2022-05-02T10:32:25     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS', 'name': 'VMSSReImageRun1SOUTHCENTRALUS', 'type': 'Microsoft.Resources/resourceGroups', 'properties': <azure.mgmt.resource.resources.v2021_04_01.models._models_py3.ResourceGroupProperties object at 0x7f9d6032bd30>, 'location': 'southcentralus', 'managed_by': None, 'tags': None}
[ 2022-05-02T10:32:27     INFO ] [ create                   ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Storage/storageAccounts/vmsspipeline9fc2a6bc', 'name': 'vmsspipeline9fc2a6bc', 'type': 'Microsoft.Storage/storageAccounts', 'tags': {'confidentiality': 'public'}, 'location': 'southcentralus', 'sku': <azure.mgmt.storage.v2021_09_01.models._models_py3.Sku object at 0x7f9d6016a490>, 'kind': 'StorageV2', 'identity': None, 'extended_location': None, 'provisioning_state': 'Succeeded', 'primary_endpoints': <azure.mgmt.storage.v2021_09_01.models._models_py3.Endpoints object at 0x7f9d6016aa30>, 'primary_location': 'southcentralus', 'status_of_primary': 'available', 'last_geo_failover_time': None, 'secondary_location': None, 'status_of_secondary': None, 'creation_time': datetime.datetime(2022, 4, 29, 16, 7, 7, 364576, tzinfo=<isodate.tzinfo.Utc object at 0x7f9d6266fdc0>), 'custom_domain': None, 'sas_policy': None, 'key_policy': None, 'key_creation_time': <azure.mgmt.storage.v2021_09_01.models._models_py3.KeyCreationTime object at 0x7f9d6016ac40>, 'secondary_endpoints': None, 'encryption': <azure.mgmt.storage.v2021_09_01.models._models_py3.Encryption object at 0x7f9d6016aa90>, 'access_tier': 'Hot', 'azure_files_identity_based_authentication': None, 'enable_https_traffic_only': True, 'network_rule_set': <azure.mgmt.storage.v2021_09_01.models._models_py3.NetworkRuleSet object at 0x7f9d6016abe0>, 'is_sftp_enabled': None, 'is_local_user_enabled': None, 'is_hns_enabled': None, 'geo_replication_stats': None, 'failover_in_progress': None, 'large_file_shares_state': None, 'private_endpoint_connections': [], 'routing_preference': None, 'blob_restore_status': None, 'allow_blob_public_access': True, 'minimum_tls_version': 'TLS1_0', 'allow_shared_key_access': None, 'enable_nfs_v3': None, 'allow_cross_tenant_replication': None, 'default_to_o_auth_authentication': None, 'public_network_access': None, 'immutable_storage_with_versioning': None, 'allowed_copy_scope': None, 'storage_account_sku_conversion_status': None, 'dns_endpoint_type': None}
[ 2022-05-02T10:32:28     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Network/networkSecurityGroups/vmssPeregrineTest-nsg', 'name': 'vmssPeregrineTest-nsg', 'type': 'Microsoft.Network/networkSecurityGroups', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'etag': 'W/"eb7d1979-ed01-4960-a05c-263594cf31d5"', 'security_rules': [<azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7f9d5a2c1a60>], 'default_security_rules': [<azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7f9d5a2c1970>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7f9d5a2c1bb0>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7f9d5a2c1be0>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7f9d5a2c1c10>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7f9d5a2c1c40>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7f9d5a2c1c70>], 'network_interfaces': None, 'subnets': [<azure.mgmt.network.v2021_05_01.models._models_py3.Subnet object at 0x7f9d5a2c1cd0>], 'flow_logs': None, 'resource_guid': 'a66db6d6-770c-4f59-9ac2-c283de4a9a24', 'provisioning_state': 'Succeeded'}
[ 2022-05-02T10:32:28     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Network/virtualNetworks/vmssPeregrineTest-vnet', 'name': 'vmssPeregrineTest-vnet', 'type': 'Microsoft.Network/virtualNetworks', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'extended_location': None, 'etag': 'W/"d5b4eb7c-ea37-462b-9ea4-fbd232834c2c"', 'address_space': <azure.mgmt.network.v2021_05_01.models._models_py3.AddressSpace object at 0x7f9d5a2c1e20>, 'dhcp_options': None, 'flow_timeout_in_minutes': None, 'subnets': [<azure.mgmt.network.v2021_05_01.models._models_py3.Subnet object at 0x7f9d6016ae20>], 'virtual_network_peerings': [], 'resource_guid': '0f1294d0-ba81-42f8-bd75-de499de66b2f', 'provisioning_state': 'Succeeded', 'enable_ddos_protection': False, 'enable_vm_protection': None, 'ddos_protection_plan': None, 'bgp_communities': None, 'encryption': None, 'ip_allocations': None}
[ 2022-05-02T10:32:28     INFO ] [ subnets_list             ] <iterator object azure.core.paging.ItemPaged at 0x7f9d6016aee0>
[ 2022-05-02T10:32:30     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Compute/virtualMachineScaleSets/vmssPeregrineTest', 'name': 'vmssPeregrineTest', 'type': 'Microsoft.Compute/virtualMachineScaleSets', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'sku': <azure.mgmt.compute.v2021_11_01.models._models_py3.Sku object at 0x7f9d5a034760>, 'plan': None, 'identity': None, 'zones': ['1'], 'extended_location': None, 'upgrade_policy': None, 'automatic_repairs_policy': None, 'virtual_machine_profile': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineScaleSetVMProfile object at 0x7f9d5a034c70>, 'provisioning_state': 'Succeeded', 'overprovision': None, 'do_not_run_extensions_on_overprovisioned_v_ms': None, 'unique_id': '37746c0b-92b4-4b45-961f-474922163bb5', 'single_placement_group': False, 'zone_balance': None, 'platform_fault_domain_count': 1, 'proximity_placement_group': None, 'host_group': None, 'additional_capabilities': None, 'scale_in_policy': None, 'orchestration_mode': 'Flexible', 'spot_restore_policy': None, 'time_created': datetime.datetime(2022, 4, 29, 16, 7, 45, 135353, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-05-02T10:32:30     INFO ] [ get_vms                  ] <iterator object azure.core.paging.ItemPaged at 0x7f9d5a2f7130>
[ 2022-05-02T10:32:30    DEBUG ] Using selector: EpollSelector
[ 2022-05-02T10:32:30     INFO ] [ get_vms                  ] <iterator object azure.core.paging.ItemPaged at 0x7f9d59fc29d0>
[ 2022-05-02T10:32:30     INFO ] Finish queuing all vms
[ 2022-05-02T10:32:32     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance1OE4NY', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d59fefd00>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d59fefee0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d59fefd90>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d59ff3070>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d59ff30a0>], 'patch_status': None}
[ 2022-05-02T10:32:32     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance1ZB63N', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d591d2100>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d591d22e0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d591d2190>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591d2430>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591d2460>], 'patch_status': None}
[ 2022-05-02T10:32:33     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance2C0PSP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d591d9520>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d591d9700>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d591d95b0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591d9850>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591d9880>], 'patch_status': None}
[ 2022-05-02T10:32:33     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance2RNIEP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d591c8ac0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d591c8ca0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d591c8a60>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591c8e20>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591c8e50>], 'patch_status': None}
[ 2022-05-02T10:32:34     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance2UT10B', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d591f18e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d591f1ac0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d591f1970>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591f1c10>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591f1c40>], 'patch_status': None}
[ 2022-05-02T10:32:35     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance3BVHI3', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d591e7e80>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d591e7cd0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d591e7dc0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591e7eb0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d591e7df0>], 'patch_status': None}
[ 2022-05-02T10:32:36     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance3NM16Y', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d5897ba90>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d5897bc70>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d5897bb20>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d5897bdc0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d5897bdf0>], 'patch_status': None}
[ 2022-05-02T10:32:36     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance46BHVP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d58983d90>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d58983f70>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d58983e20>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d58990100>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d58990130>], 'patch_status': None}
[ 2022-05-02T10:32:37     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance4MZ3H7', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d58996370>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d58996550>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d58996400>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d589966a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d589966d0>], 'patch_status': None}
[ 2022-05-02T10:32:37     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance4TUB0I', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d589a0910>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d58983280>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d589907f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d589a0a00>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d589a0a30>], 'patch_status': None}
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:32:38    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:08     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance6J1CJ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d58126a00>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d58126be0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d58126a90>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d58126d30>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d58126d60>], 'patch_status': None}
[ 2022-05-02T10:33:09     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance7LN82P', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d580bad00>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d580baee0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d580bad90>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580c7070>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580c70a0>], 'patch_status': None}
[ 2022-05-02T10:33:10     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance7T6J4T', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d5811de50>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d580fcd90>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d5811dee0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580fcee0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580fcd00>], 'patch_status': None}
[ 2022-05-02T10:33:10     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance8OF109', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d58126400>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d58126fa0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d58126130>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580cb2b0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580cb2e0>], 'patch_status': None}
[ 2022-05-02T10:33:11     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance8Y7YDL', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d591d2b20>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d580cba30>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d580cb910>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580cbb80>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580cbbb0>], 'patch_status': None}
[ 2022-05-02T10:33:11     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance91S4YT', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d580de6a0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d580de880>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d580de730>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580de9d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580dea00>], 'patch_status': None}
[ 2022-05-02T10:33:12     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance9PSKLN', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d580ba040>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d580defd0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d580ba310>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580ed160>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580ed190>], 'patch_status': None}
[ 2022-05-02T10:33:13     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceAIQZ3V', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d580edca0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d580ede80>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d580edd30>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d580edfd0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d58074040>], 'patch_status': None}
[ 2022-05-02T10:33:13     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceBEHG6E', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d58074b50>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d58074d30>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d58074be0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d58074e80>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d58074eb0>], 'patch_status': None}
[ 2022-05-02T10:33:14     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceBWHIYH', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7f9d5807ed60>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7f9d5807ef40>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7f9d5807edf0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d5808f0d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7f9d5808f100>], 'patch_status': None}
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:15    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d589a0a60>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d580e0580>, 950316.346943003)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d580cb460>
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d580b23a0>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d5811fb80>, 950316.334455942)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d580b2520>
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d580ed6d0>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d591cac40>, 950316.410035215)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d580ed640>
[ 2022-05-02T10:33:18    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-2' coro=<logging_worker() done, defined at /home/crodriguezde/experiments/reimage/imds.py:37> exception=TypeError('string indices must be integers')>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 41, in logging_worker
    logger_retry(msg)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/backoff/_sync.py", line 94, in retry
    ret = target(*args, **kwargs)
  File "/home/crodriguezde/experiments/reimage/imds.py", line 34, in logger_retry
    utils.log_imds(imds)
  File "/home/crodriguezde/experiments/reimage/utils.py", line 22, in log_imds
    logging.info('imds', extra=imds)
  File "/usr/lib/python3.8/logging/__init__.py", line 2082, in info
    root.info(msg, *args, **kwargs)
  File "/usr/lib/python3.8/logging/__init__.py", line 1446, in info
    self._log(INFO, msg, args, **kwargs)
  File "/usr/lib/python3.8/logging/__init__.py", line 1587, in _log
    record = self.makeRecord(self.name, level, fn, lno, msg, args,
  File "/usr/lib/python3.8/logging/__init__.py", line 1562, in makeRecord
    rv.__dict__[key] = extra[key]
TypeError: string indices must be integers
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d581059a0>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d580e0b20>, 950316.35845133)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d58105430>
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d580ed370>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d580f25e0>, 950316.886837162)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d580ed6a0>
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d58126f40>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d580f8040>, 950316.342748666)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d580cb550>
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d5811d2b0>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d580e0520>, 950316.350615409)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d580b2d60>
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d5811da30>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d580e0400>, 950316.339361)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d5811dc40>
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d58110ee0>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d580e0d00>, 950316.360775976)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d591b9cd0>
[ 2022-05-02T10:33:18    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f9d580fca30>
[ 2022-05-02T10:33:18    ERROR ] Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f9d58116100>, 950316.337451939)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f9d580fc4c0>
[ 2022-05-02T10:33:18     INFO ] Exit per user request
